Makefile

```diff
@@ -138,6 +156,8 @@ ${PKGDIR}/bin/gazctl: ${PKGDIR} ${GOBIN}/gazctl
 	cp ${GOBIN}/gazctl $@
 ${PKGDIR}/bin/gazette: ${PKGDIR} ${GOBIN}/gazette
 	cp ${GOBIN}/gazette $@
+${PKGDIR}/bin/flowctl-rs: ${RUSTBIN}/flowctl-rs
+	cp ${RUSTBIN}/flowctl-rs $@
```

Comment:
I just realized that I didn't have `.build/bin` in my PATH, but rather that I've always been picking up `flowctl` from my `~/go/bin`. :upside_down_face: